microsoft-edge/devtools-guide-chromium/customize/index.md

```diff
@@ -113,6 +113,37 @@ To restore the default theme, placement, tools order, and any other customized s
 See [Customize keyboard shortcuts](../customize/shortcuts.md).
 
 
+<!-- ====================================================================== -->
+## Videos about Customizing the DevTools UI
+
+
+#### Customizing Microsoft Edge Developer Tools and quick feature access
+
+_Published on 5 May 2022._
+
+[![Thumbnail image for the DevTools customization video](./index-images/customize-devtools.png)](https://www.youtube.com/watch?v=ypRzEBYNptQ)
+
+Learn how to customize DevTools to fit your needs.
+
+This video covers how to dock or undock DevTools, open new tools, and close those you don't need. It describes how to move tools in the bottom drawer and customize the text size and theme. The video also explains how you can use the Command Menu keyboard shortcuts to quickly customize DevTools.
+
+
+#### Understand the DevTools user interface
+
+_Published on 1 September 2022._
+
+[![Thumbnail image for the Understand the DevTools UI video](./index-images/learn-devtools-ui.png)](https://www.youtube.com/watch?v=ayemJLeE55c)
+
+Understand how the Microsoft Edge DevTools UI is organized. If you are confused about what tools are available, and what the main parts of the DevTools interface do, this video will help get more comfortable.
+
+In this video, we go over the UI structure, with its main toolbar and panel and its drawer area. We also go over how to find, open, and close tools, restore the default settings, and go over the list of available tools.
+
```

Comment:
@captainbrosset  - The Welcome text, not included here, includes the word 'customize'.  This section lacks the word 'customize'.  This article's title is "Customize DevTools".  Published URL:
https://docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/customize/

Welcome text:
Check out this video to learn about the main parts of the DevTools UI, how to **customize** it to meet your needs, learn about the different tools that are available, and where to go for help.

Reply:
When you read this comment thread in main page of PR, you can't see that the lines below it are additional text, from Welcome panel, that include the word 'customize'.

Reply:
Yes, I think keeping this video in this section makes sense.